test/integration_tests/test_udf_executor.py

```diff
@@ -231,6 +231,9 @@ def test_should_raise_for_udf_name_mismatch(self):
 
     def test_should_raise_if_udf_file_is_modified(self):
         execute_query_fetch_all("DROP UDF DummyObjectDetector;")
+
+        execute_query_fetch_all("DROP UDF IF EXISTS DummyObjectDetector;")
```

Comment:
Duplicate

Reply:
Added for testing `IF EXISTS`. Added a comment.